zone/entity.cpp

```diff
@@ -520,7 +520,7 @@ void EntityList::MobProcess()
 
 			old_client_count = numclients;
 
-			Spawn2* s2 = mob->CastToNPC()->respawn2;
+			Spawn2* s2 = mob->CastToNPC()->GetSpawn();
```

Comment:
This is gonna be a big concern in a hot path hitting the entity list in your ternary